### DISCUSSION TITLE
Juniper Netflow

### DISCUSSION THREAD

**Discussion by kleper** [akvorado/akvorado#1278] [Q&A]
Hi, I'm running akvorado last version.

```
akvorado_inlet_core_received_flows_total{exporter=“200."} 923822
akvorado_inlet_core_received_flows_total{exporter=“200.”} 4.474998e+06
akvorado_inlet_core_received_flows_total{exporter=“200"} 444157
akvorado_inlet_core_received_flows_total{exporter=“200”} 643773
# HELP akvorado_inlet_flow_decoder_errors_total Decoder processed error count.
```

Regarding the last log it seems the system is receiving flows but cannot decodify the flow, any ideas about this?

Thanks.

**Comment by vincentbernat** [comment]
The output you provide is either truncated or there is no errors reported.

**Comment by kleper** [comment]
I found the issue. Was a bad configuration for my side.
